examples/dt/uni01alpha/README.md

```diff
@@ -0,0 +1,317 @@
+    exit 0
+fi
+
+disks=$(lsblk -o NAME,TYPE | awk '{ if ($2 == "disk" && $1 != "sda") print "/dev/"$1}')
+disk_str=''
+
+for disk in ${disks}; do
+    pvcreate ${disk}
+    disk_str="${disk_str} ${disk}"
+done
+
+vgcreate cinder-volumes ${disk_str}
+```
+
+##### Notes
+
+The LVM backend for Cinder is a special case as the storage data is on the
+OpenShift node and has no external storage systems. This has several
+implications
+
+- To prevent issues with exported volumes, cinder-operator automatically uses
+  the host network. The backend is configured to use the host's VLAN IP
+  address. This means that the cinder-volume service doesn't need any
+  networkAttachments.
+- There can only be one node with the label `openstack.org/cinder-lvm=`. Apply
+  the label using the command
+  `oc label node <nodename> openstack.org/cinder-lvm=`
+
+### Octavia
+
+Octavia is enabled with the appropriate network attachments configured to
+deploy Octavia. It manages amphorae VMs through a self-service tenant network.
+The Octavia Amphora controllers get access to it through a Neutron externally
+routed flat provider network configured as a SNAT-less gateway for a neutron
+router linked to the tenant networks. Host routes on the tenant network's
+subnet and routes on the network attachment provide the required `next hop`
+routing to establish the necessary bidirectional routing.
+
+This arrangement requires a network attachment for connecting the OVN and
+Amphora Controller pods (octavia-housekeeping, octavia-healthmanager,
+octavia-worker). Because Neutron ML2/OVN implements provider networks by
+bridging the relevant physical interface - in this case the network-attachment,
+there is an additional requirement that this attachment function when
+bridged. As the default macvlan attachments do not function when bridged, a
+bridge network attachment is used.
+
+Bridge attachments do not directly provide connectivity outside of the OCP
+node. To implement this, the NodeNetworkConfigurationPolicy creates an VLAN
+interface as is typical for the other networks, but does not configure an IP
+pool as it is not needed. It is also not configured for metallb as it is solely
+as part of a way to establish a L2 network link between nodes. The
+NodeNetworkConfigurationPolicy also configures an octbr linux bridge which is
+configured as the bridge for the network attachment mentioned above. It is also
+configured to add the VLAN interface as a port, effectively linking the nodes
+and the network attachments.
+
+```YAML
+spec:
+  octavia:
+    enabled: true
+    template:
+      octaviaAPI:
+        networkAttachments:
+          - internalapi
+      octaviaHousekeeping:
+        networkAttachments:
+          - octavia
+      octaviaWorker:
+        networkAttachments:
+          - octavia
+      octaviaHealthManager:
+        networkAttachments:
+          - octavia
+
+  ovn:
+    template:
+      ovncontroller:
+        nicMappings:
+          - datacentre: ospbr
```

Comment:
should be:-
        nicMappings:
          datacentre: ospbr
          octavia: octbr

Reply:
Thanks @karelyatin 

Should be fixed in the next commit.